.github/workflows/deploy-preview.yml

```diff
@@ -0,0 +1,31 @@
+# .github/workflows/deploy-preview.yml
+
+name: Deploy to Preview Channel
+
+on:
+  pull_request:
+
+jobs:
+  preview:
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: ./example
+
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-java@v1

```

Comment:
Setting up Java should only be important when you are building for Android -- not targeting Web.

And if it is necessary, why version 12?  (I'm thinking that an LTS version should be used.). My guess is that you copied/pasted this from the `flutter-action` `README`.

Reply:
Fixed!